src/style.css

```diff
@@ -0,0 +1,100 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.searchfield {
+  font-family: Avenir, sans-serif;
+  box-sizing: border-box;
+  margin-top: 20px;
+  width: 100%;
+  font-weight: 300;
+  background: #fff;
```

Comment:
don't forget to create variables for all colors

Reply:
I need to create `var( )` for every color and gradient I use? Or I miss smth